addons/website/views/snippets/s_numbers_boxed.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_numbers_boxed" name="Numbers boxed">
+    <section class="s_numbers_boxed pt80 pb80 o_colored_level o_cc o_cc5">
+        <div class="container">
+            <h2 style="text-align: center;">Insights, stats, and metrics</h2>
+            <p style="text-align: center;">This section allows to highlight key statistics.</p>
```

Comment:
Because the snippet is in grid mode by default, I think the title must be in a column. In the same way, it would be if it were created from scratch.